src/Snippets/one-build.json

```diff
@@ -0,0 +1,76 @@
+{
+    "one-build": {
+        "prefix": "one-build",
+        "body": [
+            "[one-build]",
+            "one-import-tf=${1:True}",
+            "one-import-tflite=${2:False}",
+            "one-import-bcq=${3:False}",
+            "one-import-onnx=${4:False}",
+            "one-optimize=${5:False}",
+            "one-quantize=${6:False}",
+            "one-pack=${7:False}",
+            "one-codegen=${8:False}",
+            "",
+            "[one-import-tf]",
+            "input_path=$9",
+            "output_path=$10",
+            "input_arrays=$11",
+            "input_shapes=$12",
+            "output_arrays=$13",
+            "converter_version=$14",
+            "model_format=$15",
+            "save_intermediate=$16",
+            "",
+            "[one-import-tflite]",
+            "input_path=$17",
+            "output_path=$18",
+            "",
+            "[one-import-bcq]",
+            "input_path=$19",
+            "output_path=$20",
+            "input_arrays=$21",
+            "input_shapes=$22",
+            "output_arrays=$23",
+            "converter_version=$24",
+            "",
+            "[one-import-onnx]",
+            "input_path=$25",
+            "output_path=$26",
+            "input_arrays=$27",
+            "output_arrays=$28",
+            "converter_version=$29",
+            "model_format=$30",
+            "save_intermediate=$31",
+            "",
+            "[one-optimize]",
+            "input_path=$32",
+            "output_path=$33",
+            "",
+            "[one-codegen]",
+            "backend=$34",
+            "command=$35",
```

Comment:
Can you please move this to bottom side, to follow orders of `[one-build]` ?

Reply:
I revised it as you suggested. :)